core/modules/parser/QSMySqlListener.cc

```diff
@@ -1795,15 +1795,24 @@ class AggregateWindowedFunctionAdapter :
             auto starParExpr = std::make_shared<query::ValueExpr>();
             ValueExprFactory::addValueFactor(starParExpr, starFactor);
             funcExpr = query::FuncExpr::newArg1(_ctx->COUNT()->getText(), starParExpr);
-        } else if (_ctx->AVG()) {
+        } else if (_ctx->AVG() || _ctx->MAX() || _ctx->MIN() ) {
             auto param = std::make_shared<query::ValueExpr>();
             ASSERT_EXECUTION_CONDITION(nullptr != _valueFactor, "ValueFactor must be populated.", _ctx);
             ValueExprFactory::addValueFactor(param, _valueFactor);
-            funcExpr = query::FuncExpr::newArg1(_ctx->AVG()->getText(), param);
+            antlr4::tree::TerminalNode * terminalNode;
```

Comment:
antlr4::tree::TerminalNode* terminalNode;

Reply:
the LSST DM developer guide seems to indicate otherwise :-)